deploy/charts/external-secrets/values.yaml

```diff
@@ -257,6 +257,25 @@ grafanaDashboard:
   # https://github.com/grafana/helm-charts/tree/main/charts/grafana
   annotations: {}
 
+livenessProbe:
+  # -- Address for liveness probe.
+  address: ""
+  # -- Specify the maximum amount of time to wait for a probe to respond before considering it fails.
+  timeoutSeconds: 5
+  # -- Number of consecutive probe failures that should occur before considering the probe as failed.
+  failureThreshold: 5
+  # -- Period in seconds for K8s to start performing probes.
+  periodSeconds: 10
+  # -- Number of successful probes to mark probe successful.
+  successThreshold: 1
+  # -- Delay in seconds for the container to start before performing the initial probe.
+  initialDelaySeconds: 10
+  # -- Handler for liveness probe.
+  httpGet:
+    # -- Set this value to 8082 to active liveness probes.
+    port: 0
```

Comment:
Can you name this port?

Reply:
It's a values file. It's whatever you set it to after this.

Reply:
Ah hang on actually:
```yaml
                        "port": {
                            "type": "integer"
                        }
```

I need to update it to `"8082"` I guess.

Reply:
It would ignore that field.